README.md

```diff
@@ -81,27 +83,33 @@ dedicated [release](https://github.com/caas-team/sparrow/releases) can be found
 
 ### Helm
 
-Sparrow can be installed via Helm Chart. The chart is provided in the GitHub registry:
+Sparrow can be installed via Helm Chart. The chart is available in the GitHub registry:
 
 ```sh
 helm -n sparrow upgrade -i sparrow oci://ghcr.io/caas-team/charts/sparrow --version 1.0.0 --create-namespace
 ```
 
-The default settings are fine for a local running configuration. With the default Helm values, the sparrow loader uses a
-runtime configuration that is provided in a ConfigMap. The ConfigMap can be set by defining the `runtimeConfig` section.
+The default settings are suitable for a local configuration. With the default Helm values, the sparrow loader uses a checks configuration provided in a ConfigMap (the `file` loader is used). Define the `checksConfig` section to set the ConfigMap.
 
-To be able to load the configuration during the runtime dynamically, the sparrow loader needs to be set to type `http`.
+To be able to load the checks configuration during the runtime dynamically, the sparrow loader needs to be set to type `http`.
```

Comment:
The information provided here is correct and grammatically and syntactically correct.

Still, it's too much info on how to just install via helm. The readme is already too "heavy", we should place this information elsewhere. A good place would be the chart's readme, as a comment over the related entries in the values.yaml file.

Reply:
We could place several config options in the values.yaml with docu blocks

Reply:
I've put this now into the [startup section](https://github.com/caas-team/sparrow/pull/76/commits/4582c2235597918a7e73d3c2452ba4426ab3773b).